cli/src/app.rs

```diff
@@ -182,7 +182,8 @@ pub fn add_subcommands(command: Command) -> Command {
                     .short('t')
                     .long("package-type")
                     .value_name("type")
-                    .help(r#"The type of the package ("npm", "rubygems", "pypi", "maven", "nuget", "golang", "cargo")"#),
+                    .help(r#"The type of the package ("npm", "rubygems", "pypi", "maven", "nuget", "golang", "cargo")"#)
```

Comment:
Can a `.value_parser` with a `PossibleValuesParser` be used here to limit the options to valid choices like is done for the `--lockfile-type` argument of the `parse`, `analyze`, and `init` commands?

As it is now, the error is not terrible:

```
❯ target/debug/phylum package pyyaml 5.3.1 -t pype
❗ Error: HTTP request error (400 Bad Request):

The submitted package has an unsupported registry type.
```

...but it could be even better, like how `--lockfile-type` works:

```
❯ target/debug/phylum init -t corgo
error: invalid value 'corgo' for '--lockfile-type <type>'
  [possible values: yarn, npm, gem, pip, pipenv, poetry, mvn, gradle, nuget, go, cargo, spdx, auto]

  note: value 'cargo' exists

For more information, try '--help'.
```